diff --git paragraphs.field_widget.inc paragraphs.field_widget.inc
index c110062..2ecfc29 100644
--- paragraphs.field_widget.inc
+++ paragraphs.field_widget.inc
@@ -506,7 +506,14 @@ function paragraphs_field_widget_form_build(&$form, &$form_state, $field, $insta
 
         foreach (element_children($element) as $key) {
           if ($key != 'paragraph_bundle_title' && $key != 'actions' && $key != 'paragraph_bundle_preview' && $key != 'access_info') {
-            $element[$key]['#access'] = FALSE;
+            if (isset($field_state['entity'][$delta]->is_new) && $field_state['entity'][$delta]->is_new) {
+              // New paragraphs should stay on the page when collapsed to avoid
+              // losing user input, it is only hidden from display.
+              $element[$key]['#attributes'] = array('class' => array('element-invisible'));
+            }
+            else {
+              $element[$key]['#access'] = FALSE;
+            }
           }
         }
         $element['actions'] = array(
